src/gtc/cuir/cuir_codegen.py

```diff
@@ -507,6 +508,9 @@ def ctype(symbol: str) -> str:
         """
     )
 
+    def __init__(self):
+        super().__init__()
```

Comment:
@egparedes do you think the `codegen.TemplatedGenerator`s should automatically call their parent `__init__` methods?